jupyterlab_gcedetails/src/components/hardware_scaling_status.tsx

```diff
@@ -215,15 +262,21 @@ export class HardwareScalingStatus extends React.Component<Props, State> {
   }
 
   render() {
-    const { status } = this.state;
-    const progressValue = (status / 4) * 100;
+    const { status, error, instanceDetails } = this.state;
+    const progressValue = (status / 6) * 100;
```

Comment:
This is actually not a constant right? If a user isn't modifying the machine type or the GPU, this might be 1 less step? Or is that a subsequent change?

Reply:
Yeah this is not necessarily a constant if a user doesn't attach a GPU then that step will be skipped. But because the number corresponding to the Status['Complete'] is 6, the animation always works out with the complete step showing the progress bar as full. For this reason I didn't compute this value specifically for each case. But i can update it to include the case in which the attachGpu step is skipped.

Reply:
No need to add that now since we expect additional changes as part of the refactoring.